libraries/provider_rackspace_monitoring_service.rb

```diff
@@ -21,9 +21,12 @@ def whyrun_supported?
           action new_resource.package_action
         end
 
+        node.set['rackspace_monitoring']['create_entity'] = true unless node['rackspace_monitoring'].attribute?('create_entity')
+        auto_create = node['rackspace_monitoring']['create_entity'] || new_resource.create_entity ? '--auto-create-entity' : ''
```

Comment:
I'm wary of having a library set a persistent value on a node object that will be saved back to the server. I think the node object should be removed from here, and the logic should probably only depend on `new_resource.create_entity`.